python/src/addons/Index.pyx

```diff
@@ -0,0 +1,72 @@
+
+
+    def __init__(self, *args , **kwargs):
+        if (len(args)==1) and (isinstance(args[0], (bytes, unicode))):
+            directory = args[0]
+            params = {}
+        elif (len(args)==2) and (isinstance(args[0], (bytes, unicode))) and (isinstance(args[1], dict) and all(isinstance(k, (bytes, unicode)) for k in args[1].keys()) and all(isinstance(v, (bytes, unicode)) for v in args[1].values())):
+            directory = args[0]
+            params = args[1]
+        else:
+            raise Exception('can not handle type of %s' % (args,))
+
+        if isinstance(directory, unicode):
+            directory = directory.encode('UTF-8')
+
+        # inject keyvimerger
+        if not b"keyvimerger_bin" in params:
+            params[b"keyvimerger_bin"] = os.path.join(get_bin_folder(), b"keyvimerger")
```

Comment:
this allows to run keyvimerger from the python package, but the C++ side is using "keyvimerger_bin" yet (todo)